src/test/java/org/snomed/aag/rest/ServiceIntegrationControllerTest.java

```diff
@@ -378,6 +378,63 @@ void receiveCommitInformation_ShouldExpireExpectedCriteria_WhenActioningPromotio
 		assertEquals(2, pacSecondRequest.getCriteriaItems().stream().filter(CriteriaItem::isComplete).collect(Collectors.toSet()).size()); // 1 has been expired
 	}
 
+	@Test
+	void receiveCommitInformation_ShouldNotResultInExpiredTaskCriteria_WhenProjectIsPromotedToCodeSystem() throws Exception {
```

Comment:
This test is essentially the reason for the PR. 

When a project is promoted to code system, the task should not lose its completed criteria.